### DISCUSSION TITLE
Grbl Board not sending correct voltage to TTL/LSR connection.

### DISCUSSION THREAD

**Discussion by Maccas147** [arkypita/LaserGRBL#1179] [Q&A]
Hi, not sure if you can help me with this because you seem pretty knowledgeable about the subject of lasers/GRBL boards. I have a ROTTMMOTOR grbl board and MOSTIC 5.5w laser. I've updated the software on the GRBL board to Version 1.1h to get my $30-32 commands. When using lightburn for example and using the test fire option the board is only send 0.25v where I thought it should be somewhere near to the 5v mark? Any help would be appreciated if possible.

**Comment by StuartB4** [comment]
It should only send 5v if the power on the Fire button setting in Lightburn was set to 100%.

 If you have $30 set to 255 in grbl configuration that's the maximum power, if it's set to 1000 that's the maximum power. 

So basically 0 to 5 volts in 255 steps, or 0 to 5 volts in 1000 steps.
